### PR TITLE
Allow empty password files to be read

### PR DESCRIPTION
**Pull request by siladu**
It's possible to create web3 wallets with no password so it seems reasonable to allow this case

Relates to https://github.com/ConsenSys/ethsigner/issues/372

### CODE REVIEW

**Review by usmansaleem — APPROVED** [review]
LGTM.

File: signing/secp256k1/impl/src/test/java/tech/pegasys/signers/secp256k1/common/PasswordFileUtilTest.java [hunk 1]
```diff
@@ -41,19 +40,17 @@ void canReadSpaceOnlyPasswordFromFile(@TempDir final Path tempDir) throws IOExce
...
   }
 
   @Test
-  void fileWithEolOnlyThrowsException(@TempDir final Path tempDir) throws IOException {
-    final Path passwordFile = Files.write(tempDir.resolve("password.txt"), "\n".getBytes(UTF_8));
-    assertThatExceptionOfType(SignerInitializationException.class)
-        .isThrownBy(() -> PasswordFileUtil.readPasswordFromFile(passwordFile))
-        .withMessage("Cannot read password from empty file: %s", passwordFile);
+  void canReadPasswordFromEmptyFile(@TempDir final Path tempDir) throws IOException {
+    final Path passwordFile = Files.write(tempDir.resolve("password.txt"), new byte[0]);
```
Comment (jframe) [inline signing/secp256k1/impl/src/test/java/tech/pegasys/signers/secp256k1/common/PasswordFileUtilTest.java:53]: Is empty string in the above test the same as this empty file case?

Comment (siladu) [reply signing/secp256k1/impl/src/test/java/tech/pegasys/signers/secp256k1/common/PasswordFileUtilTest.java:53]: Good point :)

**Review by jframe — APPROVED** [review]

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**